jans-cedarling/cedarling/src/jwt/key_service.rs

```diff
@@ -115,8 +115,8 @@ impl KeyService {
 
 #[derive(thiserror::Error, Debug)]
 pub enum KeyServiceError {
-    #[error("Failed to decode JWK Stores from string: {0}")]
-    DecodeJwkStores(#[source] serde_json::Error),
+    #[error("Failed to decode JWK Stores from string: {0}\n{1}\n")]
```

Comment:
I don't think that it is a good idea to use `\n` in error string

Reply:
remove the linebreak here: [5ecb0c9](https://github.com/JanssenProject/jans/pull/10350/commits/5ecb0c995f0cd42950a6c6b8ffa117573ef759b4)